package.json

```diff
@@ -25,5 +26,8 @@
     "sinon-chai": "^2.10.0",
     "source-map-support": "^0.4.15",
     "webpack": "^2.5.1"
+  },
+  "dependencies": {
+    "bluebird": "^3.5.0"
```

Comment:
Yea, I think that's a very good approach and a fair enough decision.
I have only added these so we have a default dependency in order to make setup easier.
Apart from this you can still use the `configure` function to set / change these dependencies.
What do you think?